operation_test.go

```diff
@@ -2219,15 +2219,15 @@ func TestParseCodeSamples(t *testing.T) {
 
 	t.Run("Without line reminder", func(t *testing.T) {
 		comment := `@x-codeSamples`
-		operation := NewOperation(nil, SetCodeExampleFilesDirectory("testdata/code_examples"))
+		operation := NewOperation(nil, SetCodeExampleFilesDirectories("testdata/code_examples"))
 		operation.Summary = "example"
 
 		err := operation.ParseComment(comment, nil)
 		assert.Error(t, err, "no error should be thrown")
 	})
 
 	t.Run(" broken dir", func(t *testing.T) {
```

Comment:
Not that it really matters and seems to have been there before " broken dir" has a leading space in it?

Reply:
This is not my code. Forked from swaggo/swag.